components/crt_tour/crt_threads_pane_tutorial_tip/crt_threads_pane_tutorial_tip.tsx

```diff
@@ -25,18 +31,46 @@ const CRTThreadsPaneTutorialTip = () => {
         </p>
     );
 
+    const nextBtn = (): JSX.Element => {
+        return (
+            <FormattedMessage
+                id={'tutorial_tip.got_it'}
+                defaultMessage={'Got it'}
+            />
+        );
+    };
+
+    const onDismiss = (e: React.MouseEvent) => {
+        e.preventDefault();
+        const preferences = [
+            {
+                user_id: currentUserId,
+                category: Preferences.CRT_THREAD_PANE_STEP,
+                name: currentUserId,
+                value: Constants.CrtThreadPaneSteps.FINISHED.toString(),
+            },
+        ];
+        dispatch(savePreferences(currentUserId, preferences));
+    };
+
+    const overlayPunchOut = useMeasurePunchouts(['rhsContainer'], []);
+
     return (
-        <TutorialTip
+        <TourTip
+            show={true}
+            screen={screen}
             title={title}
-            singleTip={true}
+            overlayPunchOut={overlayPunchOut}
             placement='left'
+            pulsatingDotPlacement='top-start'
+            pulsatingDotTranslate={translate}
+            step={1}
+            singleTip={true}
             showOptOut={false}
-            step={Constants.CrtThreadPaneSteps.THREADS_PANE_POPOVER}
-            tutorialCategory={Preferences.CRT_THREAD_PANE_STEP}
-            screen={screen}
-            overlayClass='tip-overlay--threads-pane'
-            punchOut={useMeasurePunchoutsDeprecated(['rhsContainer'], [])}
-            autoTour={true}
+            handleDismiss={onDismiss}
+            handleNext={onDismiss}
+            interactivePunchOut={true}
+            nextBtn={nextBtn()}
```

Comment:
@codedsun Looks like this should be,
```suggestion
            nextBtn={nextBtn}
```